dev/CommonStyles/APITests/CommonStylesTests.cs

```diff
@@ -53,6 +54,58 @@ public void VerifyUseCompactResourcesAPI()
 
             MUXControlsTestApp.Utilities.IdleSynchronizer.Wait();
         }
+
+        [TestMethod]
+        public void CornerRadiusFilterConverterTest()
+        {
+            if (!PlatformConfiguration.IsOsVersionGreaterThan(OSVersion.Redstone4))
+            {
+                Log.Comment("Corner radius is only available on RS5+");
+                return;
+            }
+
+            RunOnUIThread.Execute(() => {
```

Comment:
Optional: You could just directly invoke the CornerRadiusFilterConverter directly, without the need to instantiate any UIElements or Xaml.

Reply:
Yeah I was thinking of doing that too at first, but I think we'll probably never use the converter directly in code, going through xaml is closer to our real use cases.